examples/pytorch/appnp/appnp.py

```diff
@@ -0,0 +1,68 @@
+"""
+APPNP implementation in DGL.
+References
+----------
+Paper: https://arxiv.org/abs/1810.05997
+Author's code: https://github.com/klicperajo/ppnp
+"""
+
+import torch.nn as nn
+import dgl.function as fn
+
+
+class APPNP(nn.Module):
+    def __init__(self,
+                 g,
+                 in_feats,
+                 hiddens,
+                 n_classes,
+                 activation,
+                 dropout,
+                 alpha,
+                 k):
```

Comment:
Maybe also allow the choice of using bias in linear layers as in [the original implementation](https://github.com/klicperajo/ppnp/blob/master/ppnp/pytorch/ppnp.py#L11)?

Reply:
We don't have to.  The difference of whether having a bias in a linear layer is pretty minor compared to the adjacency dropout (which is possible but not that trivial to implement)

Reply:
I agree, this is just a note.

Reply:
For now I'm going to add a comment.